ftplugin/php/mappings.vim

```diff
@@ -7,19 +7,34 @@
 "   augroup PhpactorMappings
 "     au!
 "     au FileType php nmap <buffer> <Leader>u :PhpactorImportClass<CR>
```

Comment:
I'm not even sure this will works.
This file `ftplugin/php/mappings.vim` will only be loaded for files with the `php` filetype.
So the `FileType` event might already have been triggered and the auto commands won't do a thing...

Reply:
hm .. this is just documentation not a vimscript?

Reply:
Well I feel like someone reading it might try put this in a file with the same name and path.
Since it's already somewhere else in the doc, either delete this duplicate or fix it if it's indeed broken.

Reply:
What do you mean "already somewhere else"?

We are now generating the `doc/phpactor.txt` VIM help file from the source code. These files are the canonical source of truth.

Reply:
Note that in `:help phpactor` the path of this file is not visible.

Reply:
Ok, I wasn't aware of that... * shame *
Just ignore all that :)

Reply:
It's new :) https://phpactor.github.io/phpactor/developing.html#vim-help